### ISSUE TITLE
wrong public page language

### ISSUE THREAD

**Issue by Psychoses** [AzuraCast/AzuraCast#3558]
**Intro**
Good job for many ansible, composer and file processing git issues. All is better on my install on 0.11.2 ;) I’m sorry but i will create later more issue for some other minor stuff. So keep up the great work.

**Using Docker installation method** and **Host Operating System**
i don't use docker, my env is proxmox => lxc => ubuntu 20.04 => traditional ansible install

**AzuraCast version**
0.11.2
66718829375d3827ab83cd80c67507cafa19c2e2

**Describe the bug**
On previous version of azuracast, my admin page and the public page was in french.
On current version my admin page is in french but the public page is in english.

**Expected behavior**
I want the public page to be in french like my administration page.

**Screenshots**
https://imgur.com/a/vlLVrY2

**Comment by ogrenci01** [comment]
Hi @Psychoses ;

Azuracast is open source software. Follow the Crowdin link to contribute language translations. Translation is missing for the FR language. I think the problem stems from here.

https://crowdin.com/project/azuracast

**Comment by SlvrEagle23** [comment]
@Psychoses I did find one issue where the system would mistakenly discard a locale preference from the browser if it was just a two-letter locale (i.e. `fr`) instead of a full locale string (i.e. `fr_FR`); this has been fixed as of the latest rolling-release update.

This wouldn't have been a newly introduced bug as of the latest version, though, since this issue has always existed. If the frontend components themselves aren't translating correctly, that could be a different issue entirely.
